test/case/ietf_system/user_admin.py

```diff
@@ -13,6 +15,47 @@
         target = env.attach("target", "mgmt")
         tgtssh = env.attach("target", "mgmt", "ssh")
 
+    with test.step("Verify admin user exists with default password ..."):
+        running = target.get_config_dict("/ietf-system:system")
+        users = running["system"]["authentication"]["user"]
+
+        admin = None
+        for user in users:
+            if user['name'] == 'admin':
+                admin = user
+                break
+
+        if admin is None:
+            test.fail()
+
+        if admin['password'] != "$factory$":
+            test.fail()
```

Comment:
I think this means that all flavors of Infix now has to have an admin user with the factory password in its default config, which does not seem quite right.

Maybe the test could be to add another user (`jacky`/`$factory$`), and verify that we can login as that user, with the device's default password (taken from the physical topology)? That would also avoid the test having to know about `system.json`.